client/coral-embed-stream/src/tabs/stream/containers/DraftArea.js

```diff
@@ -71,4 +77,16 @@ DraftAreaContainer.propTypes = {
   disabled: PropTypes.bool,
   rows: PropTypes.number,
   label: PropTypes.string.isRequired,
+  comment: PropTypes.object,
 };
+
+const slots = ['commentBox'];
+
+export default withFragments({
+  comment: gql`
+  fragment TalkEmbedStream_DraftArea_comment on Comment {
+    __typename
+    ${getSlotFragmentSpreads(slots, 'comment')}
+  }
```

Comment:
It would make sense to connect to `root` to, so the RTE plugin can e.g. access settings.